contracts/base/LendingPool.sol

```diff
@@ -164,6 +179,9 @@ contract LendingPool is Base, LendingPoolInterface {
         isLoan()
         whenLendingPoolNotPaused(address(this))
     {
+        // Withdraw the tokens from compound
```

Comment:
Pre-existing problem, but it seems like we should be depositing DAI tokens on liquidation, not withdrawing them.  ETH should be withdrawn and sent to liquidator.